wasmtime/_error.py

```diff
@@ -1,12 +1,16 @@
 from ctypes import byref
+import typing
+
+if typing.TYPE_CHECKING:
+    from ._ffi import P_wasmtime_error_t
 
 
 class WasmtimeError(Exception):
     def __init__(self, message):
         self.message = message
 
     @classmethod
-    def __from_ptr__(cls, ptr):
+    def __from_ptr__(cls, ptr: "P_wasmtime_error_t") -> "WasmtimeError":
```

Comment:
I'm not super familiar with this, but how come these are strings?

Reply:
Strings because they're only imported when type checking (as per TYPE_CHECKING). The imports aren't needed at run-time. The strings mean Python will ignore them at run-time. It looks at the string when it's doing static type checking though.